projects/plugins/jetpack/extensions/index.json

```diff
@@ -52,7 +52,8 @@
 		"videopress/video",
 		"ai-assistant-support",
 		"ai-assistant-form-support",
-		"paywall"
+		"paywall",
+		"contact-form"
```

Comment:
Personal pet-peeve of mine, and I know it hasn't been done for the past few blocks above, but I think those long lists are easier to parse when the items are loaded alphabetically. :)